tests/dummy/app/routes/application.js

```diff
@@ -239,8 +240,9 @@ export default Route.extend({
 
   beforeModel() {
     let tour = this.get('tour');
-
-    tour.set('autoStart', true);
+    if (config.environment !== 'test') {
```

Comment:
Did any of the tests expect it to start the tour automatically or are we clicking the start button for all of them?